modules/developing/pages/astream-functions.adoc

```diff
@@ -148,32 +148,29 @@ A function that is already running can be updated with new configuration. The fo
 
 If you need to update any other setting of the function, delete and then re-add the function.
 
-To update your function, select your function in the *Functions* dashboard. 
+To update your function, select your function in the *Functions* dashboard.
 
 image::astream-function-update.png[Update Function]
 
 . Select *Change File* to find your function locally and click *Open*.
 
-. Update your function's *Instances* and *Timeout*. When you're done, click *Update*. 
+. Update your function's *Instances* and *Timeout*. When you're done, click *Update*.
 
 . An *Updates Submitted Successfully* flag will appear to let you know your function has been updated.
```

Comment:
Is this a flag or an alert? I've heard the rectangle, green notation that appears at the top of a user's screen only referred to as an alert.